webviz_subsurface/plugins/_well_completions.py

```diff
@@ -25,6 +25,8 @@
     read_well_connection_status,
 )
 
+WELL_CONNECTION_STATUS_FILE = "share/results/tables/well_connection_status.parquet"
```

Comment:
```suggestion
WELL_CONNECTION_STATUS_FILE = Path() / "share" / "results" / "tables" / "well_connection_status.parquet"
```
and then "downcast" to simple `str` where necessary is perhaps better (compared to converting this variable to `Path` on demand).